### PR TITLE
fix: [SRM-14770]: monitored-service-summary API being called now when…

### PR DESCRIPTION
**Pull request by karansaraswat19**
… the slider is moved

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by karansaraswat19** [comment]
retrigger jest

**Comment by karansaraswat19** [comment]
retrigger cypress-cv

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/ServiceHealth.tsx [hunk 1]
```diff
@@ -186,8 +186,14 @@ export default function ServiceHealth({
                 <>
                   <ChangesSourceCard
                     monitoredServiceIdentifier={monitoredServiceIdentifier}
-                    startTime={changesTableAndSourceCardStartAndEndtime[0]}
-                    endTime={changesTableAndSourceCardStartAndEndtime[1]}
+                    startTime={
```
Comment (harshilbits) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/ServiceHealth.tsx:189]: Extract the logic of getting start time and endtime to a function which returns start and end time

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/ServiceHealth.tsx [hunk 2]
```diff
@@ -186,8 +186,14 @@ export default function ServiceHealth({
...
+                      showTimelineSlider
+                        ? (timeRange?.startTime as number)
+                        : changesTableAndSourceCardStartAndEndtime[0]
```
Comment (harshilbits) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/ServiceHealth.tsx:192]: Please cover this logic of start and endtime in a testcase.

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesSourceCard/components/ChangesSourceLoading/ChangesSourceLoading.tsx [hunk 3]
```diff
@@ -26,7 +30,11 @@ export default function ChangesSourceLoading(): JSX.Element {
 
   const loadingBlocks = useMemo(() => {
     const loadingFields: JSX.Element[] = []
-    for (let i = 1; i <= NUM_OF_LOADING_BLOCKS_TO_SHOW; i++) {
+    for (
+      let i = 1;
+      i <= (isChaosFEEnabled ? NUM_OF_LOADING_BLOCKS_TO_SHOW_WITH_CE : NUM_OF_LOADING_BLOCKS_TO_SHOW);
```
Comment (harshilbits) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesSourceCard/components/ChangesSourceLoading/ChangesSourceLoading.tsx:35]: Please cover this logic also under a testcase.

Comment (praneshg239) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesSourceCard/components/ChangesSourceLoading/ChangesSourceLoading.tsx:35]: @karansaraswat19 Please add a test case by disabling feature flag

**Review by praneshg239 — APPROVED** [review]

**Comment by karansaraswat19** [comment]
retrigger jest

**Comment by karansaraswat19** [comment]
retrigger cypress-cv

**Review by deepesh-ui — APPROVED** [review]

**Pull request merged**